avocado/core/test.py

```diff
@@ -529,6 +529,13 @@ def workdir(self):
 
     @property
     def srcdir(self):
+        """
+        This property is deprecated and will be removed in the future.
+        The :meth:`workdir` function should be used instead.
+        """
+        LOG_JOB.info("DEPRECATION NOTICE: the test's \"srcdir\" property is "
+                     "deprecated and will be removed in the future.  Please "
+                     "use the \"workdir\" property instead.")
```

Comment:
it'd be nice to include some dates/releases

Reply:
OK, using the due date on the Trello card ( https://trello.com/c/dCcxhJc2/1267-remove-testsrcdir )